Automattic/wp-calypso
# Jetpack Search: switch upsell nudge to default to monthly

#### Changes proposed in this Pull Request

* Change the upsell nudge in Settings > Performance > Search to default to monthly

#### Testing instructions

<img width="642" alt="Screen Shot 2020-11-16 at 3 15 59 PM" src="https://user-images.githubusercontent.com/820871/99314641-a65a5580-281e-11eb-9431-432d0feff208.png">


* For a wp.com and/or Jetpack site without Jetpack Search navigate to `/settings/performance/`
* Click on the upgrade notice. Verify that you are sent to the checkout with a plan that has monthly pricing.
